projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -757,6 +774,34 @@ public function inject_core_mobile_toggle() {
 		echo '<span id="wp-admin-bar-menu-toggle" style="display: none!important">';
 	}
 
+	/**
+	 * Whether the current user has indicated they want to use the wp-admin interface for the given screen.
+	 *
+	 * @param string $screen The current screen.
+	 * @return bool
+	 */
+	public function use_wp_admin_interface( $screen ) {
+		$screen_excluded = $this->is_excluded_wp_admin_interface_screen( $screen );
+		return ! $screen_excluded && 'wp-admin' === get_option( 'wpcom_admin_interface' );
+	}
+
+	/**
+	 * Returns whether we should default to wp_admin for the given screen.
+	 *
+	 * Screens that should not default to wp_admin when the wpcom_admin_interface is set.
+	 * This applies to screens that have both a wp-admin and a Calypso interface.
+	 *
+	 * @param string $screen The current screen.
+	 *
+	 * @return bool
+	 */
+	protected function is_excluded_wp_admin_interface_screen( $screen ) {
+		$excluded_screens = array(
+			'import.php',
+		);
+		return in_array( $screen, $excluded_screens, true );
+	}
```

Comment:
I didn't try it, but would it make sense or even work if we exclude also `install-plugins.php` here?

- https://github.com/Automattic/jetpack/pull/33758/files

Reply:
I tried but couldn't make it work, because `plugin-install.php` was not called.

List of `$screen`s called:

- `edit-comments.php`
- `edit-tags.php?taxonomy=category`
- `edit-tags.php?taxonomy=post_tag`
- `edit.php`
- `edit.php?post_type=jetpack-portfolio`
- `edit.php?post_type=jetpack-testimonial`
- `edit.php?post_type=page`
- `export.php`
- `import.php`
- `index.php`
- `jetpack`
- `options-discussion.php`
- `options-general.php`
- `options-reading.php`
- `options-writing.php`
- `plugins.php`
- `themes.php`
- `upload.php`
- `users.php`